wagtail/admin/static_src/wagtailadmin/js/hallo-bootstrap.js

```diff
@@ -2,8 +2,8 @@
 
 function makeHalloRichTextEditable(id, plugins) {
     var input = $('#' + id);
-    var richText = $('<div class="richtext"></div>').html(input.val());
-    richText.insertBefore(input);
+    var editor = $('<div class="halloeditor"></div>').html(input.val());
```

Comment:
Isn’t the common practice to prefix jQuery array variables with `$`? I know the rest of this function doesn’t follow this rule, so it could be nice to change other variable(s) as well.

Reply:
It is, but I don't want to refactor this code in this PR. In retrospect I probably shouldn't have renamed the variable either.